src/Editor/HoverProvider.ts

```diff
@@ -22,6 +22,8 @@ export class HoverProvider implements vscode.HoverProvider {
           {getWordRangeAtPosition: (arg0: any, arg1: RegExp) => any; getText: (arg0: any) => any;},
       _position: any, _token: any) {
     let mdfile = new vscode.MarkdownString();
+    const range = _doc.getWordRangeAtPosition(_position, new RegExp(/(.+)/g));
+    const word = _doc.getText(range);
```

Comment:
> For example, there is a way to perform the logic below when range is not undefined.
If this like is not OK with `undefined` `range` then I think there should be some guard code (like if ~~ else ~~)

Reply:
Then, when range is undefined, can I add `return null` below line 25?

Reply:
> Then, when range is undefined, can I add return null below line 25?

Yup, I think that will do...
And it would be nice to have debug message like `console.log('getWordRangeAtPosition return undefined');`
If this doesn't show too much...

for `null` , I'm not good at this, but can we use `undefined` ? I think following other return values would be safer... -_-;;;

Reply:
| And it would be nice to have debug message like console.log('getWordRangeAtPosition return undefined');
I will add it!

| for null , I'm not good at this, but can we use undefined ? I think following other return values would be safer... -_-;;;
I'll add `return new vscode.Hover(mdfile)` instead of `return null`.
An empty MarkdownString with nothing written will be returned, such as other texts that do not provide hover.

Reply:
> And it would be nice to have debug message like console.log('getWordRangeAtPosition return undefined');

I will add it!

> for null , I'm not good at this, but can we use undefined ? I think following other return values would be safer... -_-;;;

I'll add return new vscode.Hover(mdfile) instead of return null.
An empty MarkdownString with nothing written will be returned, such as other texts that do not provide hover.

Reply:
I added it!